examples/RNOneSignalTS/src/OSDemo.tsx

```diff
@@ -90,8 +93,9 @@ class OSDemo extends React.Component<Props, State> {
         const state = await OneSignal.getDeviceState();
 
         this.setState({
-            name : state.emailAddress,
-            isSubscribed : state.isSubscribed
+            // state is possibly 'null' so need to check
+            name : state? state.emailAddress : "",
```

Comment:
very minor

how about using nullish coalescence and optional chaining?


https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

```
state?.emailAddress ?? ""
```

Reply:
Good points, thanks. I'll make a followup PR for the example app.